src/Sentry.Unity.Editor/Android/DebugSymbolUpload.cs

```diff
@@ -87,6 +87,13 @@ private string _symbolUploadTaskFormat
                 stringBuilder.AppendLine(string.Empty);
                 stringBuilder.AppendLine("tasks.assembleDebug.finalizedBy sentryUploadSymbols");
                 stringBuilder.AppendLine("tasks.assembleRelease.finalizedBy sentryUploadSymbols");
+
+                if (_isExporting)
```

Comment:
Is `_isExporting` the correct condition for this? I thought this was related to `.aab` and not just the export of the Gradle project itself.

Reply:
Only if the user is exporting Android project, we need to add the additional hook for the bundle task.

Otherwise, the assemble task hook works whether .aab or .apk is built from Unity